lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/WindowsCommandBuilder.cs

```diff
@@ -48,5 +49,15 @@ protected override string GetDriverClasspath()
 
             return string.Format("'{0};'", string.Join(ClassPathSeparator, classpathList));
         }
+
+        public override string GetIpAddressFilePath()
+        {
+            return "%AZ_BATCH_JOB_PREP_WORKING_DIR%\\hostip.txt";
+        }
+
+        public override string CaptureIpAddressCommandLine()
+        {
+            return $"powershell /c \"Set-Content -Path hostip.txt -Value ((Test-Connection -ComputerName $Env:ComputerName -Count 1).IPV4Address.IPAddressToString) -NoNewline -Force\"";
+        }
```

Comment:
hostip.txt to GetIpAddressFilePath()?

Reply:
I don't understand the comment. Please elaborate.

---
In reply to: [200411313](https://github.com/sharathmalladi/reef/pull/6#discussion_r200411313) [](ancestors = 200411313)